src/Confluent.Kafka/Impl/SafeKafkaHandle.cs

```diff
@@ -258,9 +258,9 @@ internal WatermarkOffsets GetWatermarkOffsets(string topic, int partition)
             return new WatermarkOffsets(low, high);
         }
 
-        internal void Subscribe(ICollection<string> topics)
+        internal void Subscribe(IEnumerable<string> topics)
         {
-            IntPtr list = LibRdKafka.topic_partition_list_new((IntPtr) topics.Count);
+            IntPtr list = LibRdKafka.topic_partition_list_new((IntPtr) topics.Count());
```

Comment:
Quick though on it I didn't mention first: I tend to prefer using ICollection when using Count (linq count is not good on pure IEnumerable, it may just hang forever if it's an infinite enumerable for example)

Though we are enumerating just after, so it is ok in this case